build.gradle

```diff
@@ -71,34 +73,22 @@ ext {
 }
 
 group 'com.amazon.opendistroforelasticsearch'
-version = "${opendistroVersion}.1"
+version = "${opendistroVersion}.0-beta1"
```

Comment:
Just doubly checking if this artifact has to be beta

Reply:
Yes.

Its a requirement mentioned here : https://quip-amazon.com/N3IQA0zbbYnO/Plugins-check-list-in-OpenSearch-Beta1#WLX9CAdQdaS

*Plugin Artifact naming* : <plugin name>-1.0.0.0-beta1.zip

Reply:
This is a requirement for all plugins. Retaining this.